cves/CVE-2006-5752.yml

```diff
@@ -168,8 +180,8 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: yes
+    note: xss vulnerability
```

Comment:
This document may be viewed by people who do not know about cross site scripting so you should include a definition.

Reply:
added explanation.